app/controllers/miq_policy_controller/conditions.rb

```diff
@@ -157,7 +157,7 @@ def condition_build_edit_screen
 
     if params[:copy]  # If copying, create a new condition based on the original
       c = Condition.find(params[:id])
-      @condition = Condition.new(c.attributes.merge(:name => nil))
+      @condition = Condition.new(c.attributes.merge(:name => nil, :id => nil))
```

Comment:
@DNNX since this PR is merged, do you mind making a PR with your suggestion.  It does look a lot clearer.